code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -526,7 +526,7 @@
 	base_build_path = /obj/machinery/smartfridge/food
 
 /obj/machinery/smartfridge/food/accept_check(obj/item/O)
-	if(IS_EDIBLE(O))
+	if(IS_EDIBLE(O) || (istype(O,/obj/item/reagent_containers/cup/bowl) && O.reagents && O.reagents.reagent_list.len))
```

Comment:
```suggestion
	if(IS_EDIBLE(O) || (istype(O,/obj/item/reagent_containers/cup/bowl) && O.reagents?.reagent_list.len))
```

Reply:
While this would make this section slightly more efficient, I think the efficiency gain is negligible and would make the code a tad harder to read.